cmd/secrets-provider/main.go

```diff
@@ -42,6 +42,7 @@ var envAnnotationsConversion = map[string]string{
 	"DEBUG":                "conjur.org/debug-logging",
 	"JAEGER_COLLECTOR_URL": "conjur.org/jaeger-collector-url",
 	"LOG_TRACES":           "conjur.org/log-traces",
+	"JWT_TOKEN_PATH":       "conjur.org/jwt-token-path",
```

Comment:
```suggestion
	"JWT_TOKEN_PATH":       "conjur.org/jwt-token-path"
```

Reply:
In GOLANG in dicts you should put comma in the end of the final line too

Reply:
TIL